templates/baremetalset/cloudinit/userdata

```diff
@@ -7,4 +7,7 @@ users:
     ssh-authorized-keys: {{ .AuthorizedKeys }}
     sudo: ['ALL=(ALL) NOPASSWD:ALL']
     shell: /bin/bash
+runcmd:
+  - dnf install -y http://download.devel.redhat.com/rcm-guest/puddles/OpenStack/rhos-release/rhos-release-latest.noarch.rpm
```

Comment:
@stuggi private URL here. Should we make a parameter for this instead?

Reply:
yes, in the end. this is just to test/make initial deploy workflow. in the end we need to add support for rhsm/satellite based registration.